ananse/network.py

```diff
@@ -442,12 +475,9 @@ def create_expression_network(
         )
         expression[column] = np.log2(expression[column] + 1e-5)
 
-        # Create the target gene list, based on all genes
-        expression.index.rename("target", inplace=True)
-        expression = expression.reset_index()
-        expression = expression.rename(columns={column: "target_expression"})
-
         # Create the TF list, based on valid transcription factors
+        if bindingfile is None:
+            bindingfile = "/na/"
```

Comment:
what? if no filename was set, create a folder?

Reply:
:) It is a non-existing name, which will be used later to generate TFs from another source.